Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -253,6 +253,13 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
 {
   // Create event
   CLLocation *location = [locations lastObject];
+  if (_lastLocationEvent &&
+        _locationManager.desiredAccuracy == 100.0){
+  if([_lastLocationEvent[@"coords"][@"latitude"] doubleValue] != location.coordinate.latitude || [_lastLocationEvent[@"coords"][@"longitude"] doubleValue] != location.coordinate.longitude){
```

Comment:
if the `@"coords"` key is missing, this will crash the app. Most production apps I've worked on have a category on `NSDictionary` with the following method, which removes this problem effortlessly

```objc
- (id)objectForKeyNotNull:(id)key {
    id object = [self objectForKey:key];

    if (object == [NSNull null]) {
        return nil;
    }

    return object;
}
```